src/lib/y2security/security_policies/filesystem_size_rule.rb

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) [2022] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2security/security_policies/rule"
+
+module Y2Security
+  module SecurityPolicies
+    # Rule to check whether a file system is big enough
+    class FilesystemSizeRule < Rule
+      # Mount path for the file system to check
+      #
+      # @return [String]
+      attr_reader :mount_path
+
+      # Minimum size the file system should have
+      #
+      # @return [Y2Storage::DiskSize]
+      attr_reader :min_size
+
+      # @param id [String] Rule ID
+      # @param mount_path [String] Mount path for the file system to check
+      # @param min_size [Y2Storage::DiskSize] Minimum size the file system should have
+      def initialize(id, mount_path, min_size)
```

Comment:
I would use here for `min_size` keyword parameter as in future it can easily exist rule about max size or another condition on size like size range. So I would not force to always use min_size...and also calling it will look better. Compare `FilesystemSizeRule.new("id5", "/mnt", Y2Storage::DiskSize.MiB(100))` versus `FilesystemSizeRule.new("id5", "/mnt", min_size: Y2Storage::DiskSize.MiB(100))`. So as long as rule is SizeRule and not MinSizeRule I would use keyword parameter.